pom.xml

```diff
@@ -51,9 +51,7 @@
     <properties>
         <!-- Instruct the build to use only UTF-8 encoding for source code -->
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
-        <release>8</release>
+        <maven.compiler.release>11</maven.compiler.release>
```

Comment:
Thinking about it, do we actually have to do this right now? We also may interpret "moving to Java 11 as a baseline" in the sense that it is required as the Java version used to compile and run due to requirements by dependencies, but not Debezium itself. Or do you see a strong reason for doing this?